src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -1987,24 +1990,25 @@ JNI_METHOD(void, invoke)
  jint imTimeoutMs)
 {
     chip::DeviceLayer::StackLock lock;
-    CHIP_ERROR err = CHIP_NO_ERROR;
-    auto callback  = reinterpret_cast<InvokeCallback *>(callbackHandle);
-    app::CommandSender * commandSender;
-    uint32_t endpointId             = 0;
-    uint32_t clusterId              = 0;
-    uint32_t commandId              = 0;
-    jmethodID getEndpointIdMethod   = nullptr;
-    jmethodID getClusterIdMethod    = nullptr;
-    jmethodID getCommandIdMethod    = nullptr;
-    jmethodID getTlvByteArrayMethod = nullptr;
-    jobject endpointIdObj           = nullptr;
-    jobject clusterIdObj            = nullptr;
-    jobject commandIdObj            = nullptr;
-    jbyteArray tlvBytesObj          = nullptr;
-    jbyte * tlvBytesObjBytes        = nullptr;
-    jsize length                    = 0;
+    CHIP_ERROR err                     = CHIP_NO_ERROR;
+    auto callback                      = reinterpret_cast<InvokeCallback *>(callbackHandle);
+    app::CommandSender * commandSender = nullptr;
+    uint32_t endpointId                = 0;
+    uint32_t clusterId                 = 0;
+    uint32_t commandId                 = 0;
+    jmethodID getEndpointIdMethod      = nullptr;
+    jmethodID getClusterIdMethod       = nullptr;
+    jmethodID getCommandIdMethod       = nullptr;
+    jmethodID getTlvByteArrayMethod    = nullptr;
+    jobject endpointIdObj              = nullptr;
+    jobject clusterIdObj               = nullptr;
+    jobject commandIdObj               = nullptr;
+    jbyteArray tlvBytesObj             = nullptr;
+    jbyte * tlvBytesObjBytes           = nullptr;
+    jsize length                       = 0;
     TLV::TLVReader reader;
-    TLV::TLVWriter * writer = nullptr;
+    TLV::TLVWriter * writer         = nullptr;
+    uint16_t aTimedRequestTimeoutMs = static_cast<uint16_t>(timedRequestTimeoutMs);
```

Comment:
This isn't an arg, so the naming is weird...

Reply:
yeah, i have not yet figured out the better naming, put internal instead of prefix a here. thanks